src/components/QueryEditor/ServiceQueryEditor.tsx

```diff
@@ -19,36 +20,35 @@ const servicesQueryOptions: Array<SelectableValue<ServicesQueryType>> = [
   { label: 'List Service Level Objectives (SLO)', value: ServicesQueryType.listSLOs },
 ];
 
-function serviceToOption(service: Record<string, string>) {
-  return {
-    value: service,
-    label: service.Name,
-  };
-}
-
 export function ServiceQueryEditor({ query, onChange, datasource, range }: ServiceQueryEditorFormProps) {
-  const { serviceQueryType, service, region } = query;
+  //query = migrateQuery(query);
```

Comment:
Shouldn't we migrate it here too?

Reply:
removed the commented out line, i did the migation in useInitQuery instead